compose.prod.yaml

```diff
@@ -51,8 +52,12 @@ x-airflow-common:
   # In order to add custom dependencies or upgrade provider packages you can use your extended image.
   # Comment the image line, place your Dockerfile in the directory where you placed the docker-compose.yaml
   # and uncomment the "build" line below, Then run `docker-compose build` to build the images.
-   #image: ${AIRFLOW_IMAGE_NAME:-apache/airflow:2.9.3}
-  image: ${AIRFLOW_IMAGE_NAME:-suldlss/rialto-airflow:latest}
+  build: .
+  # image: ${AIRFLOW_IMAGE_NAME:-suldlss/rialto-airflow:latest} # to use a registry image, uncomment this and comment or remove
```

Comment:
this felt like the most usable/visible place for this comment, but happy to move or reformat if others think of more readable ways to explain this.

Reply:
I'd move it above the `build` element. We might not need the further comment about using a registry image since the Airflow boilerplate above also covers this.

Reply:
d'oh, i think i got boilerplate vision and totally ignored that comment right above this change.  remembered it from other airflow compose files and didn't actually read it this time.

i edited the boilerplate comments to be a little more accurate to our current setup, and made the same change in `compose.yaml` since i'd missed that on the first pass.